components/footer/Footer.tsx

```diff
@@ -0,0 +1,34 @@
+import Image from 'next/image';
+import Link from 'next/link';
+
+import { linkedin, github } from '@/public/assets-png/index';
+
+const Footer = () => {
+  const icons = [
+    {
+      src: linkedin,
+      href: 'https://www.linkedin.com/in/alexander-mclachlan-/',
+    },
+    {
+      src: github,
+      href: 'https://github.com/AlexDjangoX',
+    },
+  ];
+
+  return (
+    <footer className="flex flex-col items-center px-[1.5rem] py-[3rem] leading-[160%] text-black-400 dark:bg-black-200 dark:text-white-800 md:flex-row md:justify-between md:px-[5.3rem] md:py-[3.25rem] ">
+      <p>© 2023 Alexander Mc Lachlan. All rights reserved.</p>
+      <div className="mt-[0.5rem] flex gap-6">
+        {icons.map((icon) => (
+          <Link key={icon.src.toString()} href={icon.href} target="_blank">
```

Comment:
Generally, I would like to see a map over the icons and return an IconComponent but if you're only going to have two, I believe it's fine.